js/letters.js

```diff
@@ -16,10 +16,17 @@ var counter = {
 }
 
 function countLetters(counter, sample_text){
-  // FIX ME
-}
+  let lowerCase = sample_text.toLowerCase().replace(/[. , ' ']/g, '').split('').toString();
```

Comment:
This seems like a lot of work that gets done for every character in the sample text. Is there a way to just take one letter off at a time and then check to see if it even exists in `counter`?